Cython/Compiler/ExprNodes.py

```diff
@@ -13619,6 +13657,59 @@ def generate_operation_code(self, code, result_code,
             else:
                 code.putln(statement)
 
+    def generate_struct_code(self, operand1_name, operand2_name, operand1_type, operand2_type, op, code):
+        return '(' + (' && ' if op == '==' else ' || ').join([self.generate_ctype_code(operand1_name + '.' + member.name, operand2_name + '.' + member.name, member.type, member.type, op, code) for member in operand2_type.scope.var_entries]) + ')'
```

Comment:
Please cut long expressions like this into separate parts, explaining their meaning with variable names.

Reply:
Done.